### DISCUSSION TITLE
Ability to set composer version

### DISCUSSION THREAD

**Discussion by jkaan** [deployphp/deployer#2220] [General]
While looking through the issues I saw that there was a request for setting the composer version which was to make sure that the version of composer was consistent between deploys. https://github.com/deployphp/deployer/issues/2012

I have a problem which can also be solved by being able to set the composer version.

Composer 2.0.0 changed the output of the `vendor/composer/installed.json` file. Therefore I need to set the version to 1.10.15. Therefore I would say it's actually a good idea to be able to pass the version when installing composer. So that would look similar to this PR: https://github.com/deployphp/deployer/pull/2110.

Since I have the usecase that I actually need to set the version I think doing this would solve that problem and the previously mentioned problem too. What do you think? I can make a quick PR :).

**Comment by antonmedv** [comment]
This will be cool to have. Can you work on recipe recipes/composer.php and add composer_version parameter.

**Comment by jkaan** [comment]
@antonmedv Here's the PR: https://github.com/deployphp/deployer/pull/2222/files :)
